CONTRIBUTING.md

```diff
@@ -82,17 +83,26 @@ Your pull request should be rebased against the current master branch. Please do
 the current master branch in with your topic branch, nor use the Update Branch button provided
 by GitHub on the pull request page.
 
+## Submitting a Plugin
+
+Sharing a new plugin is one of the most impressive ways to jump into the Snap community as a contributor. This process is as simple as adding your plugin information to the [list in plugis.yml](docs/plugin.yml).
```

Comment:
This section should link to Plugin_Authoring.md for description of how to and requirements.